### ISSUE TITLE
Style/StringLiterals incorrect auto-correction

### ISSUE THREAD

**Issue by pirj** [rubocop-hq/rubocop#9393]
```yaml
# .rubocop.yml
Style/StringLiterals:
  Enabled: true
  EnforcedStyle: single_quotes
  ConsistentQuotesInMultiline: false
```

```ruby
class Foo
  def bar
    """
      Hello 1113490238402834
      World 2-3095-20395-02395-0235
      Bye 2385-0138 -013-08 1-048 1-2084 -10284-0182-0
    """
  end
end
```

```
rubocop --auto-correct --only Style/StringLiterals
```

```diff
class Foo
  def bar
-    """
+    ''"
      Hello 1113490238402834
      World 2-3095-20395-02395-0235
      Bye 2385-0138 -013-08 1-048 1-2084 -10284-0182-0
-    """
+    "''
  end
end
```

:scream: 

--------

## Expected behavior

Auto-corrects to `'''`.

## Actual behavior

Auto-corrects to `"""` to `''"`.

## Steps to reproduce the problem

See above

## RuboCop version

```
$ rubocop -V
warning: parser/current is loading parser/ruby25, which recognizes
warning: 2.5.8-compliant syntax, but you are running 2.5.7.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
1.8.1 (using Parser 3.0.0.0, rubocop-ast 1.4.0, running on ruby 2.5.7 x86_64-linux)
  - rubocop-minitest 0.10.3
  - rubocop-performance 1.9.2
  - rubocop-rails 2.9.1
  - rubocop-rspec 2.1.0
```

**Comment by pirj** [comment]
Even though the use of `''"` is not aestetically pleasant, it does the same thing.
Triple-quotes are pointless, as they do exactly the same thing as mono-quotes. However, it's a job for another (yet to be built) cop to raise such an offence.
Closing as not actionable.
